application/actions/requests_test.go

```diff
@@ -840,6 +841,63 @@ func (as *ActionSuite) Test_RequestActions() {
 	}
 }
 
+func (as *ActionSuite) Test_requestsGet() {
+	f := createFixturesForRequestQuery(as)
+
+	tests := []struct {
+		name       string
+		user       models.User
+		requestID  string
+		wantStatus int
+	}{
+		{
+			name:       "authn error",
+			user:       models.User{},
+			requestID:  f.Requests[0].UUID.String(),
+			wantStatus: http.StatusUnauthorized,
+		},
+		{
+			name:       "authz error",
+			user:       f.Users[1],
+			requestID:  f.Requests[3].UUID.String(),
+			wantStatus: http.StatusNotFound,
+		},
+		{
+			name:       "bad request ID",
+			user:       f.Users[1],
+			requestID:  "1",
+			wantStatus: http.StatusBadRequest,
+		},
+		{
+			name:       "not found",
+			user:       f.Users[1],
+			requestID:  domain.GetUUID().String(),
+			wantStatus: http.StatusNotFound,
+		},
+		{
+			name:       "no error",
+			user:       f.Users[1],
+			requestID:  f.Requests[0].UUID.String(),
+			wantStatus: http.StatusOK,
+		},
+	}
+	for _, tt := range tests {
+		as.T().Run(tt.name, func(t *testing.T) {
+			req := as.JSON("/requests/" + tt.requestID)
+			req.Headers["Authorization"] = fmt.Sprintf("Bearer %s", tt.user.Nickname)
+			req.Headers["content-type"] = "application/json"
+			res := req.Get()
+
+			body := res.Body.String()
+			as.Equal(tt.wantStatus, res.Code, "incorrect status code returned, body: %s", body)
+
+			if tt.wantStatus == http.StatusOK {
+				as.Contains(body, fmt.Sprintf(`"id":"%s"`, tt.requestID))
```

Comment:
Maybe add some more Contains with various fields, including related objects?
(If so, maybe make the conditional the opposite of what it is with a `return` in it)

Reply:
I can definitely do that. My rationale, though, was that this is specifically testing the handler. The models tests and the `convertRequest` test should verify the actual content.

Reply:
I am thinking that it would be possible that a future change to the code could sabotage the loading of the related objects in the Action and that none of the other tests would catch that.  (If they do, then that's fine.)

Reply:
I think the other tests stand a better chance of catching that. That is, unless the entire test is duplicated here.